Documentation/installation.md

```diff
@@ -2,14 +2,14 @@
 
 ## Requirements
 
- * [Cilium] Recommended: >= 1.7.0, Minimal: >= 1.6.3
+ * [Cilium] >= 1.7.0
```

Comment:
<1.8?

Reply:
yeah maybe better to limit the range to `1.7.*`

Reply:
Good point! I just back ported the commit we applied to master, but explicitly indicating we don't support Hubble 0.5 on Cilium 1.8 is a good call.